client/app/components/EventButtons.tsx

```diff
@@ -22,7 +22,8 @@ const EventButtons = ({
   const filteredCategories =
     forPage === 'competitions'
       ? eventCategories.filter((ec) => !['extremebld', 'removed'].includes(ec.value))
-      : eventCategories;
+      : forPage === 'rankings' ? eventCategories.filter((ec) => ec.value !== 'removed') 
```

Comment:
The removed category should probably just be filtered out completely out of eventCategories. Also, I just realized that Extreme BLD doesn't need to be filtered out for the competitions page at all, so all that's needed here is just to filter out 'removed' globally.